pkg/network/tracer/event.go

```diff
@@ -376,6 +386,32 @@ func newIPRouteDest(source, dest util.Address, netns uint32) *ipRouteDest {
 	return d
 }
 
+func getPortType(p uint16) network.EphemeralPortType {
+	if !ephemeralChecked {
+		procfsPath := "/proc"
+		if config.Datadog.IsSet("procfs_path") {
+			procfsPath = config.Datadog.GetString("procfs_path")
+		}
+
+		intpair := sysctl.NewIntPair(procfsPath, "net/ipv4/ip_local_port_range", 0)
+		low, hi, err := intpair.Get()
+		if nil == err {
+			ephemeralLow = uint16(low)
+			ephemeralHigh = uint16(hi)
+			log.Infof("got ephemeral port %v %v", ephemeralLow, ephemeralHigh)
+		} else {
+			log.Infof("failed to get ephemerail %v", err)
```

Comment:
typo: `ephemerail`

Reply:
Should this be warn level?

Reply:
should be nothing; I inadvertently left some printf debugging.  Removed.